src/rma/js/components/destruction-list/create-modal.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import Modal from 'react-modal';
+
+import { CsrfInput } from "./csrf";
+import { Input, TextInput } from "./inputs";
+import { SelectInput } from "./select";
+import { countObjectKeys } from "../../utils";
+
+
+function CreateModal(props) {
+    const { checkboxes, modalIsOpen, setIsOpen, reviewers, url, csrftoken } = props;
```

Comment:
Arrow functions are considered best practice afaik, and you can already destructure the props:

```suggestion
const = CreateModal = ({ checkboxes, modalIsOpen, setIsOpen, reviewers, url, csrftoken }) => {
```